pontoon/base/static/css/fontawesome-all.css

```diff
@@ -13,7 +13,11 @@
   font-style: normal;
   font-variant: normal;
   text-rendering: auto;
-  line-height: 1; }
+  line-height: 1;}
+
+.fab-gh {
+  width: 1.8em;
+}
```

Comment:
This is a CSS template by Font Awesome, so we don't edit it. Let's revert this change.

Reply:
Noted working on this.